pkgs/development/libraries/gstreamer/default.nix

```diff
@@ -9,42 +9,42 @@
 }:
 
 let
-  callGstreamerPackage =
-    path: attrs:
-    (callPackage path attrs).overrideAttrs (oldAttrs: {
-      buildInputs = oldAttrs.buildInputs ++ lib.optionals stdenv.hostPlatform.isDarwin [ apple-sdk_13 ];
-    });
+  gstreamer-apple-sdk = apple-sdk_13;
 in
 {
-  gstreamer = callGstreamerPackage ./core { };
+  inherit stdenv;
```

Comment:
This should be `apple-sdk_gstreamer` a la `apple-sdk_swift` as it is the version of `apple-sdk` used by GStreamer, and we should `inherit` it in the scope (instead of `stdenv`, which we have no reason to inherit now) rather than passing it explicitly to all the packages. Other than that LGTM with the commits squashed.